e2e/pages/calendar.po.ts

```diff
@@ -0,0 +1,27 @@
+import { by, element, browser, ExpectedConditions, promise } from 'protractor';
+import { Wait } from '../enums/wait';
+
+export class CalendarPage {
+  private calendarEntryClass = by.className('fc-v-event');
+
+  getNumberOfVisibleEvents(): promise.Promise<number> {
+    this.quickWaitToRenderEvents();
+    return element.all(this.calendarEntryClass).count();
+  }
+
+  quickWaitToRenderEvents() {
+    // Due some reasons events in calendar aren't displayed until some action will be taked
+    // workaround for it is to select already selected view mode (month/week/day/list)
```

Comment:
might be connected to the initialized binding in our calendar component, that it is fired before ucCalendar is assigned, but I may be wrong here.
The thing is, that this issue seemed to be resolved long time ago.